src/main/java/duke/commands/Deadline.java

```diff
@@ -0,0 +1,16 @@
+package duke.commands;
+
+public class Deadline extends Task{
+
+    protected String deadlineDate;
+
+    public Deadline(String description, String deadlineDate) {
+        super(description);
+        this.deadlineDate = deadlineDate;
+    }
+
+    @Override
+    public String toString() {
+        return "[D] " + super.toString() + " (by:" + deadlineDate + ")";
```

Comment:
Consider refactoring the '[D]' into a constant/enum so that it is easier to change in the future.

Reply:
Yep, I've changed that already